Added export of `CPM_LAST_PACKAGE_NAME` (#403)
cpm-cmake/CPM.cmake: README.md

- `<dependency>_SOURCE_DIR` is the path to the source of the dependency.
- `<dependency>_BINARY_DIR` is the path to the build directory of the dependency.
- `<dependency>_ADDED` is set to `YES` if the dependency has not been added before, otherwise it is set to `NO`.

> - CPM_LAST_PACKAGE_NAME is set to `<dependency>`. For single-argument syntax this will be the automatically determined name.

Reviewer: Rephrased to be more clear imo and formatted CMake identifiers as code.

```suggestion
- `CPM_LAST_PACKAGE_NAME` is set to the determined name of the last added dependency (equivalent to `<dependency>`).
```

Author: Agreed, much better 💯
